docs/source/admin/quick_howto/cachegroup_fallback_qht.rst

```diff
@@ -0,0 +1,65 @@
+..
+..
+.. _cachegroup-fallback-qht:
+
+******************************
+Configure CacheGroup Fallbacks
+******************************
+
+.. seealso:: :ref:`tp-configure-cache-groups`
+
+#. Go to 'Topology', click on Cache Groups, and click on your desired Cache Group or click the + button to create a new Cache Group.
```

Comment:
Can you make all instances of 'Cache Group' in the text into terms, so that they link to the glossary? You don't need to worry about it in section titles, and I'm not gonna make you do it in the API docs - there are a lot of instances in the API that need to be updated and they aren't the responsibility of this PR. They'll be done in a more general-purpose PR.

Example:
```rst
This links to the glossary: :term:`Cache Group`
Terms need to be followed by a space or punctuation, so to make it plural you need to do: :term:`Cache Group`\ s
```

Reply:
done